pathology/multiple_instance_learning/panda_mil_train_evaluate_pytorch_gpu.py

```diff
@@ -540,5 +464,85 @@ def main_worker(gpu, args):
     print("ALL DONE")
 
 
+def parse_args():
```

Comment:
I think maybe it's not necessary to change the function position?

Thanks.

Reply:
I think for the overall readability, it's better this way. Thanks